cmd/immutest/command/init.go

```diff
@@ -72,7 +72,7 @@ func Init(cmd *cobra.Command, o *c.Options) {
 		}
 		defer cl.disconnect(cmd, nil)
 		serverAddress := cl.immuClient.GetOptions().Address
-		if serverAddress != "127.0.0.1" && serverAddress != "localhost" {
+		if serverAddress != "127.0.0.1" && serverAddress != "0.0.0.0" && serverAddress != "localhost" {
```

Comment:
We need this? We can have local connection also with 0.0.0.0

Reply:
I thought we should keep also the check fir `127.0.0.1` just in case the user explicitly configures it as such.